StyleCop.Analyzers/StyleCop.Analyzers/Helpers/SpecializedTasks.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Threading.Tasks;
+
+    internal static class SpecializedTasks
+    {
+        internal static Task CompletedTask { get; } = Task.FromResult(default(VoidResult));
```

Comment:
`Task.CompletedTask` not supported in portable subset.
We can add one more rule for internal analyzer #813 :smile: